challenge.rb

```diff
@@ -5,8 +5,8 @@
 # Complete the method `is_of_age?`. It should accept either strings or numbers in its arguments,
 # and should return either `true` or `false` based on whether `age` is less than `minimum_age`.
 
-def is_of_age? age, minimum_age
-  # Replace this comment with your code.
+def is_of_age?(age, minimum_age)
+    return age < minimum_age
```

Comment:
Correct, but keep in mind that 'return' is not required in Ruby; it will simply return the last evaluated expression.